test.mjs

```diff
@@ -27,5 +28,24 @@ function testContext({any, html, css, gql, md}, name) {
 			assert.equal(gql`a${'b'}c${1}`, 'abc1');
 			assert.equal(md`a${'b'}c${1}`, 'abc1');
 		});
+
+		it('strings with escapes', () => {
+			assert.equal(any`\\\na${'\\\na'}`, '\\\na\\\na');
+			assert.equal(html`\\\na${'\\\na'}`, '\\\na\\\na');
```

Comment:
Why not assert  `html`, `css`  ... functions self is the same, so we only need  test one function.

```js
assert.equal(html, any);
assert.equal(css, any);
...
```